content/en/blog/2023/announcing-notation-rc5.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Notation v1.0.0-RC.5 is available!
+author:  "Notation Release Team"
+date: 2023-05-18
+draft: false
+---
+
+The Notation maintainers are pleased to announce v1.0.0-RC.5 for [notation](https://github.com/notaryproject/notation) and [notation-go](https://github.com/notaryproject/notation-go). This blog walks you through the major updates of this release.
+
+## Major updates in notation and notation-go
+
+Notation is a CLI tool to sign and verify OCI artifacts. The v1.0.0-RC.5 changes include but not limited to the following items. See the [v1.0.0-RC.5  release notes](https://github.com/notaryproject/notation-go/releases/tag/v1.0.0-rc.5) for details.
+
+* Improve output message when there is no signature associated
+* Comply with the [OCI Distribution Spec 1.1.0-RC.2](https://github.com/opencontainers/distribution-spec/releases/tag/v1.1.0-rc2) and [OCI image-spec v1.1.0-rc.3](https://github.com/opencontainers/image-spec/releases/tag/v1.1.0-rc.3)
+* Use SHA2 instead of insecure SHA1 for certificate thumbprint in `notation inspect`
```

Comment:
It's SHA256 as SHA2 is a set of hashes such as SHA224, SHA256, SHA384, SHA512, and other variants.
```suggestion
* Use SHA256 instead of insecure SHA1 for certificate thumbprint in `notation inspect`
```